pkg/gofr/datasource/arango/arango.go

```diff
@@ -0,0 +1,322 @@
+package arango
```

Comment:
Package and folder should be renamed


```suggestion
package arangodb
```

https://github.com/gofr-dev/gofr/pull/1429#discussion_r1931755022

Reply:
done